acid/templates/base.html

```diff
@@ -42,6 +42,19 @@
                 <li class="nav-item {% if request.path.startswith('/builds') %}active{% endif %}">
                     <a class="nav-link" href="{{ url_for('builds.show_builds_history') }}">Builds</a>
                 </li>
+                <li>
+                    <div class="col ml-5"></div>
+                </li>
+                <li class="nav-item ml-3">
```

Comment:
Maybe it would be a good idea to move this checkbox to a specific page (i. e. Status) or create a template similar to base template that would be included in features that require auto-refresh? Also I'd move it from nav-bar to body of the page.

Reply:
IMO navbar is a good place for this button and it should be always visible even if not necessary. 
But I agree that when I am on subpage like history, I don't want my browser to refresh because it is pointless.

Reply:
after speaking with PO we gonna make toggle visible only when its nessesary.